apps/www/_blog/2024-12-19-how-to-hack-the-base.mdx

```diff
@@ -10,7 +10,7 @@ categories:
 tags:
   - launch-week
   - security
-date: '2024-12-20'
```

Comment:
Changing the date of the article here, because this article was published on the 19th PT. 
https://github.com/supabase/supabase/pull/31235